cmd/tailscale/backend.go

```diff
@@ -81,6 +81,11 @@ var googleDNSServers = []netip.Addr{
 // VPN status was revoked.
 var errVPNNotPrepared = errors.New("VPN service not prepared or was revoked")
 
+// errMultipleUsers is used when we get a "INTERACT_ACROSS_USERS" error, which
+// happens due to a bug in Android. See:
+//    https://github.com/tailscale/tailscale/issues/2180
+var errMultipleUsers = errors.New("VPN cannot be created on this device due to an Android bug with multiple users")
```

Comment:
Include some short link users can go to for more info?

Like `tailscale.com/s/multi-user-bug` ?

That can just be a www redirects.txt entry to some open or closed bug in oss.

Reply:
Whoops, I meant to put this comment on the const multipleUsersText below, not this error.

Reply:
Done in https://github.com/tailscale/tailscale-www/pull/1610 - good idea